src/views/Admin.js

```diff
@@ -0,0 +1,380 @@
+const rowSelection = {
+  onChange: (selectedRowKeys, selectedRows) => {
+    console.log(
+      `selectedRowKeys: ${selectedRowKeys}`,
+      'selectedRows: ',
+      selectedRows
+    );
+  },
+  getCheckboxProps: (record) => ({
+    disabled: record.name === 'Disabled User', // Column configuration not to be checked
+    name: record.name,
+  }),
+};
+
+function onChange(pagination, filters, sorter, extra) {
+  console.log('params', pagination, filters, sorter, extra);
+}
+
+function Admin() {
+  const [selectionType, setSelectionType] = useState('checkbox');
+  let history = useHistory();
+  let [query, setQuery] = React.useState('');
+  return (
+    <Layout style={{ backgroundColor: '#fff' }}>
+      <Row justify="start" align="middle">
+        <Col span={3}>
+          <a href="/" style={{ margin: '15px' }}>
+            <img src="/logo.png" width={'160px'} />
+          </a>
+        </Col>
+        <Col span={17}>
+          <Breadcrumb style={{ marginLeft: '20px' }}>
+            <Breadcrumb.Item>
+              <a href="/">Home</a>
+            </Breadcrumb.Item>
+            <Breadcrumb.Item>
+              <a href="">User Name</a>
+            </Breadcrumb.Item>
+            <Breadcrumb.Item>Administration</Breadcrumb.Item>
+          </Breadcrumb>
+        </Col>
+        <Col span={4}>
+          {/* configure displaying logout button */}
+          <LoginButton />
+        </Col>
+      </Row>
+
+      <Layout>
+        <Affix offsetTop={60}>
+          <Sider width={250}>
+            <Menu
+              mode="inline"
+              theme="light"
+              defaultOpenKeys={['users', 'resources']}
+              style={{ height: '100%', borderRight: 0 }}
+            >
+              {/* <Menu.Item>
+                <h1 style={{ fontSize: '1.5em' }}>Actions</h1>
+              </Menu.Item> */}
+              <Menu.Item
+                key="dashboard"
+                icon={<AreaChartOutlined />}
+                style={{ marginTop: '30px' }}
+              >
+                Dashboard
+              </Menu.Item>
+              <SubMenu key="users" title="Users" icon={<TeamOutlined />}>
+                <Menu.Item key="11">User Type</Menu.Item>
+                <Menu.Item key="12">Organizations</Menu.Item>
+                <Menu.Item key="13">Administrators</Menu.Item>
+              </SubMenu>
+              <SubMenu
+                key="resources"
+                title="Resources"
+                icon={<FileProtectOutlined />}
+              >
+                <Menu.Item key="21">Resource Type</Menu.Item>
+                <Menu.Item key="22">Pending Approval</Menu.Item>
+              </SubMenu>
+            </Menu>
+          </Sider>
+        </Affix>
+        <Content
+          style={{
+            padding: '24px 24px 24px',
+            marginTop: '10px',
+          }}
+          offsetTop={100}
+        >
+          <Card id="overview" style={{ marginBottom: '20px' }}>
```

Comment:
Can you turn theses Cards into their own subcomponents (you can keep them in the same file).

It'll make the JSX a bit more readable and make it the code just a bit more reusable.